### PR TITLE
Issue 3162 - Update Tabs across Teamspace in v5

### LINKED ISSUE #3162: Update tabs across Teamspace in v5
Updated design
image

Existing build
image

- [x] Remove Page title
- [x] Change case of tab titles to title case
- [x] Add box-shadow under tab bar and remove border
- [x] Selected state should include _primary.main_ text colour
- [x] Tab titles should be 13px (may need to increase to 14px if they look too small in the browser)
- [x] Height of tab bar should be 50px
- [x] First tab should always remain left aligned to the left of the 'Home' icon as shown in updated design

### PR DESCRIPTION
**Pull request by Amantini1997**
This fixes #3162 

#### Description
Design adjusted according to mockup

### CODE REVIEW

**Review by george-hadfield — DISMISSED** [review]

**Review by sanmont3drepo — APPROVED** [review]

File: frontend/src/v5/ui/routes/dashboard/projects/header/header.styles.ts [hunk 1]
```diff
@@ -19,20 +19,12 @@ import styled from 'styled-components';
...
-	display: flex;
-	flex-direction: column;
-	justify-content: space-between;
-	width: 100%;
-	max-width: 1289px;
-	margin: 28px auto 0;
+	/* 14px for home-icon left-padding + left-margin */
+	/* 15px breadcrumbs left-margin */
+	/* 30px 3D logo width */
+	/* 20px header left-padding */
```
Comment (sanmont3drepo) [inline frontend/src/v5/ui/routes/dashboard/projects/header/header.styles.ts:28]: If we are not using these styles, just delete theme

**Review by sanmont3drepo — CHANGES_REQUESTED** [review]

File: frontend/src/v5/ui/routes/dashboard/projects/header/header.styles.ts (hunk 1 above)
Comment (Amantini1997) [reply frontend/src/v5/ui/routes/dashboard/projects/header/header.styles.ts:28]: deleted

**Review by sanmont3drepo — APPROVED** [review]

**Pull request merged**